<launch>
    <node name="ocr_node" pkg="jetson_camera" type="ocr_img_processor.py" output="screen" />
</launch>
